src/MB_Toolbox.php

```diff
@@ -201,6 +201,97 @@ public function createDrupalUser($user) {
   }
 
   /**
+   *
+   */
+  public function createNorthstarUser($user) {
+
+    $northstarAPIConfig = $this->mbConfig->getProperty('northstar_config');
+    if (empty($northstarAPIConfig['host'])) {
+      throw new Exception('MB_Toolbox->createNorthstarUser() northstar_config missing host setting.');
+    }
+
+    // Required - at least one of email or mobile must be set.
+     $requiredSet = false;
+    if (isset($user->email)) {
+      $post['email'] = $user->email;
+      $requiredSet = true;
+    }
+
+    if (isset($user->mobile)) {
+      $post['mobile'] = $user->mobile;
+      $requiredSet = true;
+    }
+
+    if (!$requiredSet) {
+      throw new Exception('MB_Toolbox->createNorthstarUser() Neither email or mobile value available. Failed to create user.');
+    }
+
+    // Optional fields that can be a part of a Northstar user document
+    // List of supported fields:
+    // https://github.com/DoSomething/northstar/blob/dev/documentation/endpoints/users.md#create-a-user
+    $supportedFields = [
+      'password',
+      'first_name',
+      'last_name',
+      'addr_street1',
+      'addr_street2',
+      'addr_city',
+      'addr_state',
+      'addr_zip',
+      'country', // two character country code
+      'language',
+      'source',
+      'race',
+      'religion',
+      'college_name',
+      'degree_type',
+      'major_name',
+      'hs_gradyear',
+      'hs_name',
+      'sat_math',
+      'sat_verbal',
+      'sat_writing',
+    ];
+
+    foreach($supportedFields as $field) {
+      if (isset($user->$field)) {
+        $post[$field] = $user->$field;
+      }
+    }
+
+    // Optional fields that require formatting
+    if (isset($user->birthdate) && strpos($user->birthdate, '/') > 0 && strtotime($user->birthdate) != FALSE) {
```

Comment:
You don't need to compare statement to `!= FALSE`, that's what if statement does by default.